plugins/woocommerce/tests/e2e-pw/tests/merchant/products/block-editor/enable-block-product-editor.spec.js

```diff
@@ -0,0 +1,66 @@
+const { test, expect } = require( '@playwright/test' );
+const {
+	clickAddNewMenuItem,
+	isBlockProductEditorEnabled,
+	toggleBlockProductEditor,
+} = require( '../../../../utils/simple-products' );
+
+const ALL_PRODUCTS_URL = 'wp-admin/edit.php?post_type=product';
+const NEW_EDITOR_ADD_PRODUCT_URL =
+	'wp-admin//admin.php?page=wc-admin&path=%2Fadd-product';
```

Comment:
Double `/` in this URL as well

Reply:
Nice catch @lanej0! I fixed it in the commit [3f877ef](https://github.com/woocommerce/woocommerce/pull/39463/commits/3f877ef7e5d1b1394d5961729b23794e2c064efb)
Could you take another look at this PR?